eng/dcppack/dcppack.csproj

```diff
@@ -39,8 +39,16 @@
 
   <Target Name="Build" />
 
+  <Target Name="AddPackageFiles" BeforeTargets="_GetPackageFiles">
+    <ItemGroup>
+      <DcpFiles Include="$(NuGetPackageRoot)microsoft.developercontrolplane.$(DcpPlatform)/$(MicrosoftDeveloperControlPlanedarwinamd64PackageVersion)/tools/**/*" />
```

Comment:
NIT: Can we use the GeneratePackagePath property instead? Or does that not work when using PackageDownload?

Reply:
It doesn't seem to work for `PackageDownload`.